conf/application.conf

```diff
@@ -19,6 +19,7 @@ appName="gform-frontend"
 formMaxAttachmentSizeMB = 10
 #Below defines what kind of attachment can be uploaded to gform (file-upload)
 contentTypesSeparatedByPipe = "application/pdf|image/jpeg|application/vnd.openxmlformats-officedocument.spreadsheetml.sheet|application/vnd.oasis.opendocument.spreadsheet|application/vnd.openxmlformats-officedocument.wordprocessingml.document|application/vnd.oasis.opendocument.text|application/vnd.openxmlformats-officedocument.presentationml.presentation|application/vnd.oasis.opendocument.presentation"
+restrictedFileExtensionList = ["JFIF", "PNG"]
```

Comment:
it looks like these values need to always in uppercase for it to work. Using CIString for storing config value solves the problem